bundles/org.openhab.binding.oppo/README.md

```diff
@@ -221,7 +221,8 @@ sitemap oppo label="Oppo Blu-ray" {
         Setpoint item=oppo_sub_shift label="Sub Title Shift [%d]" minValue=-10 maxValue=10 step=1 visibility=[oppo_power==ON]
         Selection item=oppo_hdmi_mode visibility=[oppo_power==ON] icon="video"
         Selection item=oppo_hdr_mode visibility=[oppo_power==ON] icon="colorwheel"
-        Selection item=oppo_remote_button visibility=[oppo_power==ON]
+        // Selection item=oppo_remote_button visibility=[oppo_power==ON]
```

Comment:
Here and in other README's: Why is this commented out? IMHO either it should be kept as is or removed. Since it's an example configuration, it's okay to provide more than one example.

Reply:
Having the selection with the same items as the button grid seemed redundant. But then I considered that Button grid is does not work on the iOS app nor OH versions <4.1.0. So I decided to comment out the deprecated examples instead of removing them completely. I could add a note for each similar to the tivo example. WDYT?

Reply:
The tivo example looks great. 👍 I would still suggest to either completely remove the examples that are truly deprecated. If they could still make sense for some use-cases/inspiration, they could also be left there (without being commented).

Reply:
OK I noted the deprecated elements instead of commenting them out. The only thing commented is the Roku TV example that is separate from the Roku player. I suppose I could make a separate sitemap example for the TV.

Reply:
Or you can just uncomment the last example to avoid the redundancy of having two complete sitemaps?

Reply:
done!